src/main/java/com/faforever/client/game/CreateGameController.java

```diff
@@ -386,13 +387,30 @@ public void onCreateButtonClicked() {
         .map(ModVersion::getUid)
         .collect(Collectors.toSet());
 
+    int minRating = 0;
+    int maxRating = 3000;
```

Comment:
Nowhere should constants be found that imply a max or a min rating.... There is no max and no min rating. If no max rating is set then there is no max rating....